projects/1.20/assets/modrinth-deepdrilling/deepdrilling/lang/zh_cn.json

```
@@ -0,0 +1,63 @@
+{
+  "block.deepdrilling.andesite_drill_head": "安山岩制钻头",
+  "block.deepdrilling.asurine_node": "皓蓝石矿脉",
+  "block.deepdrilling.asurine_node.tooltip.behaviour1": "_皓蓝石，凝灰岩，方解石_",
+  "block.deepdrilling.asurine_node.tooltip.condition1": "表面沉积物",
+  "block.deepdrilling.asurine_node.tooltip.summary": "在_寒冷生物群系_的_基岩层_中生成",
+  "block.deepdrilling.blank_module": "空白组件",
```

```suggestion
  "block.deepdrilling.blank_module": "空白模块",
```
不是，那里按组件翻是因为原文是component，没有模块module
其他的module就是module，按模块翻

---

行